.travis.yml

```diff
@@ -36,6 +36,9 @@ jobs:
     - stage: readme
       python: 2.7
       env: TOXENV=readme
+    - stage: coveralls
+      python: 3.7
+      env: TOXENV=coveralls
```

Comment:
It should upload to coveralls after running (test) jobs.
With this it will have a separate (new) job that would run tests, and upload coverage for this only.

Reply:
Do you mean upload on each test job?

Reply:
Yes, but not for qa ones etc.
Coveralls allows for multiple/parallel uploads itself.
If codecov would be used it could have different flags per job.

Reply:
That's a nice feature in codecov, indeed. I don't have a strong opinion about coveralls, so we could use codecov instead.